routes/api/v1/forecast.js

```diff
@@ -0,0 +1,45 @@
+var express = require("express");
+var router = express.Router();
+var User = require('../../../models').User;
+var Forecast = require('../../../pojos/forecast');
+const bcrypt = require('bcrypt');
+require('dotenv').config()
+const fetch = require('node-fetch')
+
+router.get("/", function (req, res) {
+  if(req.body.api_key){
+    user =  User.findOne({
+        where: {
+          api_key: req.body.api_key
+        }
+      })
+      .then(user => {
+        if (!user){
+          res.status(409).send()
+        } else {
+  let location = req.query.location
+  fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${location}&key=${process.env.GOOGLE_KEY}`)
```

Comment:
These calls to Google Geocode and Dark Sky API's look good and "DO THE THING!" :)
If there's time before delivering this, it'd be a fun spot to come back to refactor and abstract to their own services for re-usability.